mandatory/2-function-creation.js

```diff
@@ -4,7 +4,15 @@
     1. the user should be 18 or older
     2. the user must be logged in
 */
-function isAcceptableUser(userAge, isLoggedIn) {}
+
+function isAcceptableUser(userAge, isLoggedIn) {
+  
+if(userAge>=18 && isLoggedIn===true){
```

Comment:
===true is not necessary there .
if (userAge >= 18 && isLoggedIn ) does the same work